include/secp256k1.h

```diff
@@ -553,7 +553,7 @@ SECP256K1_API int secp256k1_ecdsa_signature_serialize_compact(
 /** Verify an ECDSA signature.
  *
  *  Returns: 1: correct signature
- *           0: incorrect or unparseable signature
+ *           0: incorrect or unparsable signature
```

Comment:
That one is correct, see https://en.wiktionary.org/wiki/unparseable

Reply:
@real-or-random thanks for your review. I haved reverted it.